comic_identity_engine/database/repositories.py

```
@@ -0,0 +1,703 @@
+        series_run = SeriesRun(
+            title=title,
+            start_year=start_year,
+            publisher=publisher,
+        )
+        self.session.add(series_run)
+        try:
+            await self.session.flush()
+            await self.session.refresh(series_run)
+            logger.info(f"Created series run: {series_run}")
+        except sqlalchemy_exc.SQLAlchemyError as e:
+            logger.error(f"Error creating series run: {e}")
```

Ruff (0.15.2)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)